modules/run-consul/run-consul

```diff
@@ -203,6 +203,9 @@ function generate_supervisor_config {
   local readonly consul_bin_dir="$5"
   local readonly consul_user="$6"
 
+  local consul_user_home_dir=""
+  consul_user_home_dir="$(get_owner_home_dir $consul_user)"
```

Comment:
This code format is necessary because the following code will not exit the script upon a failure of `get_owner_home_dir`:

```bash
# Script won't exit on a non-zero exit code from get_owner_home_dir
local readonly consul_user_home_dir="$(get_owner_home_dir $consul_user)"
```